sdk/inc/azure/core/az_http.h

```diff
@@ -182,12 +185,13 @@ AZ_NODISCARD AZ_INLINE az_result az_http_response_init(az_http_response* respons
  *
  * @see https://tools.ietf.org/html/rfc7230#section-3.1.2
  */
+// Member order is optimized for alignment.
```

Comment:
Just double check that this isn't visible in the generated docs.

Reply:
No, it is not, and that's intentional.